app/actions/remote/entry/gql_common.ts

```diff
@@ -265,6 +266,12 @@ export const entry = async (serverUrl: string, teamId?: string, channelId?: stri
         result = entryRest(serverUrl, teamId, channelId, since);
     }
 
+    // Fetch data retention policies
+    const isDataRetentionEnabled = await getIsDataRetentionEnabled(database);
+    if (isDataRetentionEnabled) {
+        fetchDataRetentionPolicy(serverUrl);
+    }
```

Comment:
When the user first login, `isDataRetentionEnabled` will always be false as the `result` retuned is used to store data in the db, including the config and license has not been stored yet.

Reply:
https://github.com/mattermost/mattermost-mobile/blob/b191154db94f0c3e4d1eaa9d32d6e861de1a5c7c/app/actions/remote/entry/login.ts#L46-L53

By the time it reaches the code, we would have config and license in the DB right?